src/datasources/cache/cache.router.ts

```diff
@@ -403,4 +432,30 @@ export class CacheRouter {
   static getExchangeRatesCacheDir(): CacheDir {
     return new CacheDir(CacheRouter.EXCHANGE_RATES_KEY, '');
   }
+
+  static getNativeCoinPriceCacheDir(args: {
+    nativeCoinId: string;
+    fiatCode: string;
+    tokenAddress?: string;
```

Comment:
This argument can be removed.

Reply:
Fixed in [d4cf6711](https://github.com/safe-global/safe-client-gateway/commit/d4cf6711d2b6f399acd83fd4be5b8ff10e1b778d)